common/config.js

```diff
@@ -0,0 +1,113 @@
+/*
+A half-way implementation between just a json `require('config.json')` and using the node module `require('config')`
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const ENV_MOCK='mock';
+const ENV_DEV='dev';
+const ENV_PROD='prod';
+const ENV_CROMWELL_DEV='cromwell-dev';
+
+const marthaEnv = process.env.ENV || ENV_DEV;
+const terraEnv =
+    (() => {
+        // noinspection JSUnreachableSwitchBranches
+        switch (marthaEnv) {
+        case ENV_MOCK: return 'dev';
+        case ENV_CROMWELL_DEV: return 'dev';
+        default: return marthaEnv;
+        }
+    })();
+
+// Start with the defaults...
+const configDefaults = {
+    samBaseUrl:
+        `https://sam.dsde-${terraEnv}.broadinstitute.org`,
+    bondBaseUrl:
+        (() => {
+            // noinspection JSUnreachableSwitchBranches
+            switch (marthaEnv) {
+            case ENV_MOCK: return 'http://127.0.0.1:8080';
+            default: return `https://broad-bond-${terraEnv}.appspot.com`;
```

Comment:
nit: indent?

Reply:
Existing lint configuration said no. I'll take a really quick look and see if I can find the eslint rule to fix this though.

Reply:
Found it!